public/en/platform/protect-sdk-php/action-documentation.md

```diff
@@ -6,6 +6,13 @@ The purpose of the Action Client is to enable calls to NS8 vi the API for managi
   * The ability to pass new, unknown (to the SDK) events to the API for processing
   * Interaction with the NS8 API without specifically utilizing HTTP components or needing to customize request logic
 
+### Actions VS Events
```

Comment:
Might be worth linking to the overall docs:

* https://github.com/ns8inc/protect-integration-docs/blob/dev/public/en/platform/protect-integration-docs/switchboards_foundational_switches.md
* https://github.com/ns8inc/protect-integration-docs/blob/dev/public/en/platform/protect-integration-docs/switchboards_actions_v_events.md